dubbo-admin-server/src/main/java/org/apache/dubbo/admin/controller/ServiceController.java

```diff
@@ -126,4 +126,10 @@ public ServiceDetailDTO serviceDetail(@PathVariable String service, @PathVariabl
     public Set<String> allApplications(@PathVariable String env) {
         return providerService.findApplications();
     }
+
+    @RequestMapping(value = "/consumers", method = RequestMethod.GET)
+    public Set<String> allConsumers(@PathVariable String env) {
+        List<Consumer> consumers = consumerService.findAll();
```

Comment:
Is the `consumers` will be null?

Reply:
No, if the consumer is empty `consumers` is an empty list.
https://github.com/apache/dubbo-admin/blob/2b04f1adfdfacab3ac20114bdb4b239672772ab8/dubbo-admin-server/src/main/java/org/apache/dubbo/admin/common/util/SyncUtils.java#L94-L99